Nondeterministic build failure TestCallingRPCFromRPC

```
--- FAIL: TestCallingRPCFromRPC (0.41s)

	rpc_test.go:373: expected FOO RPC to be called

FAIL

FAIL	github.com/NebulousLabs/Sia/modules/gateway	10.686s
```

--- a/modules/gateway/rpc_test.go
+++ b/modules/gateway/rpc_test.go
@@ -502,13 +502,13 @@ func TestCallingRPCFromRPC(t *testing.T) {
 		if err != nil {
 			t.Fatal(err)
 		}
-	case <-time.After(200 * time.Millisecond):
+	case <-time.After(500 * time.Millisecond):
 		t.Fatal("expected FOO RPC to be called")
 	}
 
 	select {
 	case <-barChan:
-	case <-time.After(200 * time.Millisecond):
+	case <-time.After(500 * time.Millisecond):
 		t.Fatal("expected BAR RPC to be called")
 	}
 }

